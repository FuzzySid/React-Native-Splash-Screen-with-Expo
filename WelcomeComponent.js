import React from 'react';
import {Text} from 'react-native';

export default function FirstComponent(){
    return(
        <Text>Welcome! ðŸ‘‹</Text>

    )
}