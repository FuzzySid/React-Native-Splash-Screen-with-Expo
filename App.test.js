import React from 'react';
import renderer from 'react-test-renderer';

import WelcomeComponent from './WelcomeComponent';

describe('<WelcomeComponent />', () => {
  it('Gets Rendered', () => {
    const _WelcomeComponent = renderer.create(<WelcomeComponent />);
    const welcomeComponentTree=_WelcomeComponent.toJSON();
    expect(welcomeComponentTree).toMatchSnapshot();
  });
});