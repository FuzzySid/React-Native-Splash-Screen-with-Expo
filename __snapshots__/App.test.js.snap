// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomeComponent /> Gets Rendered 1`] = `
<Text>
  Welcome! ðŸ‘‹
</Text>
`;
